<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>processwindow</class>
 <widget class="QWidget" name="processwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableWidget" name="ProcessInfo">
        <column>
         <property name="text">
          <string>ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>进程名称</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>状态</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>内存(页)</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>CPU(%)</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton_UndoSpare">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>31</height>
            </size>
           </property>
           <property name="text">
            <string>UndoSpare</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_SpareOneCore">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>31</height>
            </size>
           </property>
           <property name="text">
            <string>SpareOneCore</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_kill">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>31</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>31</height>
            </size>
           </property>
           <property name="text">
            <string>结束进程</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>进程数：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>运行数：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>睡眠数：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>130</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>僵死数：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_pNum">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_pRun">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_pSleep">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>90</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_pZombie">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>130</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
